pkg/backup/alter_backup_schedule_test.go

```diff
@@ -194,3 +194,88 @@ func TestAlterBackupScheduleSetsIncrementalClusterID(t *testing.T) {
 	)
 	require.Len(t, rows, 2)
 }
+
+func TestAlterBackupScheduleDoesNotResumePausedSchedules(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	th, cleanup := newAlterSchedulesTestHelper(t, nil)
+	defer cleanup()
+
+	t.Run("standalone paused full backup is not resumed", func(t *testing.T) {
+		rows := th.sqlDB.QueryStr(t,
```

Comment:
nit: this is a cleaner way to grab the schedule id:
```
var scheduleID int
QueryRow(stmt).Scan(&scheduleID)
```

Reply:
If I'm not mistaken, `QueryRow().Scan` requires you to provide a location for every column in the row or else it errors. Although tbh I only did it this way because I copied it from above.